### PR TITLE
Min should and match phrase support

### PR DESCRIPTION
**Pull request by theus77**

### CODE REVIEW

File: Entity/Form/Search.php [hunk 1]
```diff
@@ -353,4 +360,22 @@ public function setContentType($contentType)
         $this->contentType = $contentType;
         return $this;
     }
+
+    /**
+     * @return int
```
Comment (dameert) [inline Entity/Form/Search.php]: Redundant comment as return type is defined allready

File: Entity/Form/Search.php [hunk 2]
```diff
@@ -353,4 +360,22 @@ public function setContentType($contentType)
...
+     */
+    public function getMinimumShouldMatch(): int
+    {
+        return $this->minimumShouldMatch;
+    }
+
+    /**
+     * @param int $minimumShouldMatch
+     * @return Search
```
Comment (dameert) [inline Entity/Form/Search.php]: Redundant comment block

File: Entity/Form/Search.php (hunk 1 above)
Comment (theus77) [reply Entity/Form/Search.php]: Solved, but I would like to find a way to disable itnfrom php storm? https://www.jetbrains.com/help/phpstorm/settings-file-and-code-templates.html?

Comment (dameert) [reply Entity/Form/Search.php]: I think the very last point in this article might help https://www.jetbrains.com/help/phpstorm/phpdoc-comments.html

**Pull request merged**